### PR TITLE
KAHZ Q330/3 to Centaur datalogger swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removing Q330/3 (952) from Kahuranaki
- Replacing with Nanometrics Centaur (9895)
- Using latest Centaur firmware version 4.8.5
- Streams affected EH-ENZ

### CODE REVIEW

File: install/dataloggers.csv [hunk 1]
```diff
@@ -993,6 +993,7 @@ Nanometrics,Centaur CTR4-6S,9889,Paritu Road,,2023-06-26T20:28:00Z,9999-01-01T00
 Nanometrics,Centaur CTR4-6S,9891,Cape Kidnappers,,2023-05-25T04:20:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,9893,Porangahau,,2023-05-04T02:09:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,9894,"Amberley, Maskells Rd",,2023-06-27T00:27:00Z,9999-01-01T00:00:00Z
+Nanometrics,Centaur CTR4-6S,9895,Kahuranaki,,2023-07-19T23:31:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline install/dataloggers.csv:996]: Opening time is 2023-07-19T**21**:31:00Z?

**Review by mnaguit — CHANGES_REQUESTED** [review]
just a quick change in Centaur opening time, to match firmware, thanks @Thomas-Benson 👍

File: install/dataloggers.csv (hunk 1 above)
Comment (Thomas-Benson) [reply install/dataloggers.csv]: My bad, fixed now.

**Review by CBurton90 — APPROVED** [review]

**Review by mnaguit — APPROVED** [review]
👍

**Pull request merged**